scripts/inpage-bridge/src/index.js

```diff
@@ -1,7 +1,8 @@
-/* global inpageBundle */
+
+import inpageProvider from './provider';
```

Comment:
nit. do you have thoughts on changing this name to be a little more descriptive? 

```suggestion
import initializeInpageProvider from './provider';
```

Reply:
Since we're adding new properties to the `window` object, does `injectInpageProvider` sound more accurate?

Reply:
works for me